src/scaraplate/rollup.py

```diff
@@ -158,6 +160,29 @@ def get_target_project_cookiecutter_context(
         return dict(context)
 
 
+def ensure_cookiecutter_context_exists(
```

Comment:
It seems to me, or is this logic a little duplicate of `get_target_project_cookiecutter_context`?

Reply:
Not exactly. Both these functions just wrap `cookiecutter_context.read()` and provide a meaningful output depending on the result. And the output in both cases is different.

For example, `get_target_project_cookiecutter_context` prints the context, but we don't need that in `ensure_cookiecutter_context_exists`.

Reply:
ok